ros/src/twist_controller/dbw_node.py

```diff
@@ -81,20 +81,38 @@ def loop(self):
 
             if all([self.twist_cmd, self.current_velocity, self.dbw_enabled]):    # Ensure values have been initialized
 
+                ########## for testing purposes only, do not integrate into master ####################
```

Comment:
Is this section supposed to be removed?

Reply:
It can be removed or commented out. It may be helpful for quickly debugging longitudinal controller.